pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -378,7 +379,7 @@ profiles:
 	}
 }
 
-func TestCodecsEncodePluginConfig(t *testing.T) {
+func TestCodecsEncodePluginConfigv1beta1(t *testing.T) {
```

Comment:
Why can't we use the same function for v1beta1 and v1beta2?

Reply:
We can, I just segregated the tests

Reply:
That just adds duplicated code. You can maybe put a comment before starting the v1beta2 tests, but I don't think it's necessary.

Reply:
This has been done.